course/grading.py

```diff
@@ -88,7 +88,7 @@ def get_prev_visit_grades(
 
 
 @course_view
-def get_prev_grades_dropdown_content(pctx, flow_session_id, page_ordinal):
+def get_prev_grades_dropdown_content(pctx, flow_session_id, ordinal):
```

Comment:
I think I would've preferred standardizing the other way. `ordinal` is less descriptive than `page_ordinal`.

Reply:
Ok. Is it necessary to standardize that name in model?  (needs migrations).

Reply:
Updated, wth `FlowPageData.ordinal` renamed to `page_ordinal`.

Reply:
:/